grammars/rust.cson

```diff
@@ -322,11 +322,23 @@
   }
   {
     'comment': 'Function call'
-    'match': '\\b([a-zA-Z_][a-zA-Z0-9_]*)\\s*\\('
+    'match': '\\b([a-zA-Z][a-zA-Z0-9_]*)\\s*\\('
```

Comment:
A function may start with `_`.
The correct regex is `\\b([A-Za-z][A-Za-z0-9_]*|_[A-Za-z][A-Za-z0-9_]*)`